release-controller/ci_check.py

```diff
@@ -35,66 +35,80 @@ def success_print(message: str):
     print(f"{Fore.GREEN}{message}{Fore.RESET}")
 
 
-def error_print(message: str):
-    print(f"{Fore.RED}{message}{Fore.RESET}")
+def print_and_ret(message: str) -> str:
+    print(message)
+    return message
 
 
-def warn_print(message: str):
-    print(f"{Fore.YELLOW}{message}{Fore.RESET}")
+def error_print(message: str) -> str:
+    return print_and_ret(f"{Fore.RED}{message}{Fore.RESET}")
```

Comment:
I think it would be useful to print a prominent `ERROR:` or similar, to make it clear which line was an error. Color is great (let's keep it), but sometimes we're primed for letters, and may get confused about what the error actually was.

Reply:
Added:
```
✅ SUCCESS: Schema validation passed
❌ ERROR: Schema validation failed
```